.github/CODEOWNERS

```diff
@@ -459,7 +459,6 @@ app/services/form1010cg/service.rb @department-of-veterans-affairs/vfs-10-10 @de
 app/services/form1010_ezr_attachments/file_type_validator.rb @department-of-veterans-affairs/vfs-10-10 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/services/hca/overrides_parser.rb @department-of-veterans-affairs/vfs-10-10 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/services/identity @department-of-veterans-affairs/octo-identity
-app/services/identity/account_creator.rb @department-of-veterans-affairs/octo-identity
```

Comment:
This is the only line I actually deleted. This line is covered by the line above it, so it's not needed.

All other "deletions" are actually moves.